quadruped_pympc/controllers/Controllers.py

```diff
@@ -0,0 +1,543 @@
+import os
+import time
+
+# TODO: Ugly hack so people dont have to run the python command specifying the working directory.
+#  we should remove this before the final release.
+import sys
+
+# Add the parent directory of this script to the Python path
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import mujoco
+import numpy as np
+
+
+# Parameters for both MPC and simulation
+from quadruped_pympc.helpers.foothold_reference_generator import FootholdReferenceGenerator
+from quadruped_pympc.helpers.periodic_gait_generator import PeriodicGaitGenerator
+from quadruped_pympc.helpers.swing_trajectory_controller import SwingTrajectoryController
+from quadruped_pympc.helpers.terrain_estimator import TerrainEstimator
+from quadruped_pympc.helpers.quadruped_utils import GaitType
+from gym_quadruped.utils.quadruped_utils import LegsAttr
+
+
+
+
+
+####
+    
+#------------
+class Controller:
```

Comment:
I would use the name `SRBController` or `SingleRigidBodyController`, to clearly state to the user what type of control its executed.